pkg/dashboard/templates/image-scan-details.gohtml

```diff
@@ -3,70 +3,94 @@
 
 <head>
     {{ template "head" . }}
-    <link rel="stylesheet" href="static/css/scan-details.css">
+  <link rel="stylesheet" href="static/css/scan-details.css">
 </head>
 
 <body>
 <div class="navbar">
   <div class="navbar-content">
     <a href="/">
-      <img class="logo" src="static/images/polaris-logo.png" alt="Polaris" />
+      <img class="logo" src="static/images/polaris-logo.png" alt="Polaris"/>
     </a>
   </div>
 </div>
 
 <div class="main-content">
   <div class="card details">
-    <span> {{ .ImageTag }} </span>
-    <p> {{ .ScanResult }} </p>
+    <h3 style="float:right;" class="image-result-{{ .ScanResult }}"> {{ .ScanResult }} </h3>
+    <h1>{{ .ImageTag }}</h1>
```

Comment:
`ImageTag` might be veeeeery long, which might break formatting. Maybe we could add some shortening logic?

For example, from original full-image-tag `container-registry-name/module-name/service-name:branch_githash_build_number` leave only `service-name:{tag.subsctring(15)}`

Reply:
we can use css attribute `text-overflow`, i will apply this.
>         overflow: hidden;
>         text-overflow: ellipsis;

Reply:
<img width="934" alt="Screen Shot 2019-12-11 at 15 42 03" src="https://user-images.githubusercontent.com/784409/70622353-d2438300-1c2c-11ea-8a61-26064da28ccd.png">